### DISCUSSION TITLE
图标模糊问题

### DISCUSSION THREAD

**Discussion by likemuuxi** [tw93/Pake#458] [Q&A]
为什么 npm run dev 调试模式图标很清晰，npm run build 打包后图标变得模糊
![image](https://github.com/tw93/Pake/assets/85612715/853cdd49-556f-4e29-8f34-4a893f5a6e7e)

**Comment by tw93** [comment]
这个我没有Windows的机器，可以看看是不是图片本身的质量问题，你用pake默认的图片试试
